client/cli.rs

```diff
@@ -26,6 +26,10 @@ pub enum SubCommand {
         /// Delays enqueueing the task until <delay> elapses. See enqueue for accepted formats.
         #[structopt(name = "delay", short, long, conflicts_with = "immediate", parse(try_from_str=parse_delay_until))]
         delay_until: Option<DateTime<Local>>,
+
+        /// Only start the task after <after> task has finished
```

Comment:
`after <after>` is somewhat confusing.
Maybe
`/// Only start the task after all specified tasks have finished.`

+ If we decide to auto fail the task if a parent task fails, this should be mentioned in this doc string as well.

Reply:
Updated to : 
`Start the task once all specified tasks have successfully finished. If one of the dependencies failed, this task will be marked as failed`